SPECS/cmake/cmake.spec

```
@@ -61,6 +64,7 @@ sed -i -e "s|@@CMAKE_VERSION@@|%{version}|" -e "s|@@CMAKE_MAJOR_VERSION@@|%{majo
 # Should be removed once the issue is fixed upstream and we apply the fix: https://gitlab.kitware.com/cmake/cmake/-/issues/22470.
 rm -f %{_lib64dir}/lib{stdc++,gfortran}.a
 
+export JAVA_HOME=/usr/lib/jvm/msopenjdk-17
```

Would this work:
```suggestion
export JAVA_HOME="%{java_home}"
```

I think this was a way to avoid hard-coding a specific version but I'm still trying to find the place where this macro is defined.

---

Ah, it's inside `javapackages-tools`, which should be available -it's pulled in by `BuildRequires: javapackages-generators`.

---

I see `javapackages-tools` is still on `msopenjdk-11`, so this may not work after all. Unless we also update that package.

---

good idea. it does look like it works, and javapackages-tools is already updated for version 17